workflow/setup_expt.py

```diff
@@ -219,7 +219,7 @@ def link_files_from_src_to_dst(src_dir, dst_dir):
     src_dir = os.path.join(inputs.icsdir, current_cycle_dir, src_atm_anl_dir)
     dst_dir = os.path.join(comrot, current_cycle_dir, dst_atm_anl_dir)
     makedirs_if_missing(dst_dir)
-    for ftype in ['abias', 'abias_pc', 'abias_air', 'radstat']:
+    for ftype in ['abias', 'abias_pc', 'abias_air', 'radstat', 'atminc.nc']:
```

Comment:
This won't be enough for hybrid-4DEnVar (you'll likely also need atmi003.nc and atmi009.nc) but this is fine for ATM 3DVar.

Reply:
Good point @CoryMartin-NOAA . We only tested 3D IAU and warm restarts.

Reply:
This file `atminc.nc` is not going to be available when doing a atm-only, 3DVar.  In which case this will be silently ignored.

Most experiments at resolutions /= ops, will start with a cold-start of the atmosphere.  I would suggest the S2S application should also start with a cold-start atmosphere at lower resolutions for consistency, and maintainability.  

If a warm-start of the atmosphere is desired at lower resolutions, we will need programs that can create warm-start ics from higher resolutions, as well as increments at the resolutions.

To support a warm-start with IAU, will require significant work in the configuration system.  This has been identified as a gap in the global-workflow planning for SFS applications.  I am not sure accumulating further technical debt for a single purpose is the way to go forward.

I am happy to discuss this capability and support of this requirement offline, but as it stands now, this is adding a special case that adds complexity and burden to the configuration system without a plan.

Reply:
We already had that discussion offline @aerorahul . We need to start from warm start IC's, that's what that PR is addressing. Sorry for not being clear. 
I don't think `atminc.nc` is silently ignored, it will fail if not present.

Reply:
It does not fail.  It is ignored.
Please checkout the branch associated with this PR and do the following:
```bash
$> ./workflow/setup_expt.py gfs cycled --app ATM --pslot 1944atmcyc --start cold --idate 2021122018 --edate 2021122106 --gfs_cyc 0 --resdet 96 --resens 48 --nens 2 --expdir /scratch1/NCEPDEV/stmp2/Rahul.Mahajan/EXPDIR --comrot /scratch1/NCEPDEV/stmp2/Rahul.Mahajan/ROTDIR --icsdir /scratch1/NCEPDEV/global/glopara/data/ICSDIR/C96C48
EDITED:  /scratch1/NCEPDEV/stmp2/Rahul.Mahajan/EXPDIR/1944atmcyc/config.base as per user input.
EDITED:  /scratch1/NCEPDEV/stmp2/Rahul.Mahajan/EXPDIR/1944atmcyc/config.aeroanl as per user input.
EDITED:  /scratch1/NCEPDEV/stmp2/Rahul.Mahajan/EXPDIR/1944atmcyc/config.landanl as per user input.
EDITED:  /scratch1/NCEPDEV/stmp2/Rahul.Mahajan/EXPDIR/1944atmcyc/config.ocnanal as per user input.
```
This is so because on L225, there is a check for the file to exist.  If it is not present, it will skip it.  That is a silent ignore.

Reply:
Also, if the `--start=warm` option is turned on, we don't know at this this of `setup_expt.py`, whether the user desires to setup a 3D or 4D experiment.  There are manual changes required in `config.base` after the `setup_expt.py` step is performed which will address which increment files (middle, left, right); 3-hrly increment, hourly-increment, etc.
So, none of this is/can be captured in the current configuration system.  Therefore, any attempt to get this specific configuration option in, will open the door for other "specific" requirements without properly addressing the configuration gap.  
I hope this puts some light on the reluctance on my part to including this.
If this is a requirement from the application teams, we can accelerate this development, but that will come at the expense of something else.  Feel free to discuss with the teams.